src/extensions/sonarcloud/tasks/SonarCloudAnalyze/v2/task.json

```diff
@@ -10,7 +10,7 @@
   "version": {
     "Major": 2,
     "Minor": 0,
-    "Patch": 0
+    "Patch": 1
```

Comment:
when we bump default scanner versions, is it patch or minor?

Reply:
Yeah, I wasn't sure about this, IMO we're not adding any feature from the user POV so it's a patch? But I don't have a strong opinion on this

Reply:
I was thinking maybe because the scanner-cli version is a major bump it might warrant a bit more "caution". I'm fine either way